CHANGELOG.md

```diff
@@ -1,4 +1,19 @@
 ## 1.10.0 (Unreleased)
+
+FEATURES
+
+* provider/scaleway: update terraform SDK to 0.12.0 [GH-127]
```

Comment:
why `[GH-127]` and  not `#127`?

Reply:
I follow the [best practices format](https://www.terraform.io/docs/extend/best-practices/versioning.html#entry-format)